cmd/cored/disable_mockhsm.go

```diff
@@ -1,22 +1,17 @@
-//+build prod
+//+build disable_mockhsm
```

Comment:
I like this. How about instead of `plain_http` we go with `no_https_redirect`. This would leave us with the default behavior of: disabled loopback authn, mockhsm, enabled reset endpoints and https redirects. The build tags would be:
```
loopback_authn
no_mockhsm
no_reset
no_https_redirect
```